superset/databases/commands/update_ssh_tunnel.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
lets make a sub directory for ssh tunnel
`superset/databases/ssh_tunnel/update.py`